src/scenario/actiontype/actiontype_change_allowed_buildings.c

```diff
@@ -0,0 +1,20 @@
+#include "actiontype_change_allowed_buildings.h"
+
+#include "scenario/data.h"
+#include "building/menu.h"
+
+void scenario_event_action_change_allowed_buildings_init(scenario_action_t *action)
```

Comment:
For all of these empty inits.
I was not sure if keeping the coding consistent was more important than avoiding unneeded function calls.
(I found it easier to code in a consistent way.)
If needed I can easily remove these.